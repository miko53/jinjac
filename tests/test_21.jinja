{{ [ 1, 2, 3, 4 ] }}
{{ ( 8, 9, 40, 5.5, 10.2 ) }}
{{ [ 1, name, 5, 10 ] }}
{{ [ 150, 'test', 5.5 ] }}


