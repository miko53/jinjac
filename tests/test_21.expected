[1, 2, 3, 4]
(8, 9, 40, 5.500000, 10.200000)
[1, 'mickael', 5, 10]
[150, 'test', 5.500000]


